Make all_parents_must_converge settable when creating node

fosterseth opened the pull request:
##### SUMMARY
<!--- Describe the change, including rationale and design decisions -->
#7063 

When targeting the `api/v2/workflow_job_templates/id#/workflow_nodes/` endpoint, user could not set `all_parents_must_converge` to true.

this change fixes that.

<!---
If you are fixing an existing issue, please include "related #nnn" in your
commit message and your description; but you should still explain what
the change does.
-->

##### ISSUE TYPE
<!--- Pick one below and delete the rest: -->
 - Bugfix Pull Request

##### COMPONENT NAME
<!--- Name of the module/plugin/module/task -->
 - API

##### AWX VERSION
<!--- Paste verbatim output from `make VERSION` between quotes below -->
```
awx: 11.2.0
```

ryanpetrello commented on awx/api/serializers.py:
```diff
@@ -3600,7 +3600,7 @@ def validate(self, attrs):
         if ujt is None:
             ret = {}
-            for fd in ('workflow_job_template', 'identifier'):
+            for fd in ('workflow_job_template', 'identifier', 'all_parents_must_converge'):
```
Could we add a functional/API test for this?

ryanpetrello replied:
Maybe under `awx/main/tests/functional/api/test_workflow_node.py` ?

fosterseth replied:
added a functional test

fosterseth committed:
Make all_parents_must_converge settable when creating node

When targeting, ../workflow_job_templates/id#/workflow_nodes/ endpoint,
user could not set all_parents_must_converge to true.

awx issue #7063
